### PR TITLE
added displayMonthTextStyle

### PR DESCRIPTION
**Pull request by Mayb3Nots**
Added option to customise the displayMonth text

### CODE REVIEW

File: lib/flutter_neat_and_clean_calendar.dart [hunk 1]
```diff
@@ -370,7 +370,7 @@ class _CalendarState extends State<Calendar> {
             children: <Widget>[
               todayIcon ?? Container(),
               Text(
-                displayMonth,
+                displayMonth.toUpperCase(),
```
Comment (rwbr) [inline lib/flutter_neat_and_clean_calendar.dart:373]: Please remove the `.toUpperCase` there. This changes the behavior of the library. I don't think, everybody likes to show up the months in uppercase letters.

**Comment by rwbr** [comment]
Thank you for your contribution. I would like to merge it into the main branch. However, I don't want to include the `toUppercase` method call in line 373 of flutter_neat_and_clean_calendar.dart. This line should be reverted to its original state.

**Comment by Mayb3Nots** [comment]
> Thank you for your contribution. I would like to merge it into the main branch. However, I don't want to include the `toUppercase` method call in line 373 of flutter_neat_and_clean_calendar.dart. This line should be reverted to its original state.

Ah yea that wasn't supposed to be in the PR

**Comment by Mayb3Nots** [comment]
new and updated PR #29

**Pull request closed without merging**